var/spack/repos/builtin/packages/py-tomli/package.py

```diff
@@ -12,8 +12,8 @@ class PyTomli(PythonPackage):
     Tomli is fully compatible with TOML v1.0.0."""
 
     homepage = "https://github.com/hukkin/tomli"
-    url = "https://files.pythonhosted.org/packages/py3/t/tomli/tomli-1.2.2-py3-none-any.whl"
-    list_url = "https://pypi.org/simple/tomli/"
+    pypi = "https://pypi.org/project/tomli/"
```

Comment:
Was this change intentional? I don't think `pypi` supports wheels at the moment, you have to use `url`. Either way, I would save these changes for a separate PR since they aren't related to yt.

Reply:
Changed back to `pypi`